STYLEGUIDE.md

```diff
@@ -45,10 +45,7 @@ This part contains style guide rules around general (framework agnostic) HTML us
 Use camel case for the values of attributes such as `id` and `data-test-subj` selectors.
 
 ```html
-<button
-  id="veryImportantButton"
-  data-test-subj="clickMeButton"
->
+<button id="veryImportantButton" data-test-subj="clickMeButton">
```

Comment:
My IDE automatically linted this file. I think it's fine to check it in like this but let me know if you want me to revert it.